templates/puppetagent.conf.erb

```diff
@@ -53,9 +53,12 @@
 <% if @http_proxy_port != 'UNSET' -%>
     http_proxy_port = <%= @http_proxy_port %>
 <% end -%>
-    report = true
+    report = <%= @puppet_reports %>
     graph = true
     pluginsync = true
+    interval = <%= @puppet_run_interval %>
+    splay = true
```

Comment:
This changes the default behavior without any way to override. It should be a parameter.